xero_accounting.yaml

```diff
@@ -27994,10 +27994,10 @@ components:
       - IGDSINPUT3
       - SROVR
       - TOURISTREFUND
-      - TXRCN33INPUT
-      - TXRCREINPUT
-      - TXRCESSINPUT
-      - TXRCTSINPUT
```

Comment:
@dev-welli do you have more context from when these were added 9 months ago?

Reply:
See below comments but TLDR; When adding the new 23 tax rates we took the opportunity to fix the existing rates which are currently wrongly affixed with input to the end.